docs/src/_getting-started/installation.md

```diff
@@ -10,27 +10,30 @@ Before you can use Meltano, you’ll need to get it installed. We have a [comple
 
 ## Install Meltano
 
+Use the following command to check that you have a supported Python version installed:
 
+```bash
+python --version
+```
 
-{% tabs log %}
+Meltano supports all versions of Python that have not yet reached their end-of-life date. Refer to [the official Python documentation for when versions of Python will reach end-of-life](https://devguide.python.org/versions/).
+
+<div class="notification is-warning">
+  <p>Not all plugins support the latest versions of Python.</p>
+</div>
 
-{% tab log Using Pip %}
+{% tabs log %}
 
-Meltano is pip-installable.
+{% tab log Using `pipx` %}
 
-Use the following command to check that you have a supported Python version installed:
+Since Meltano is an application, it should always be installed into a clean virtual environment without any other packages installed alongside it.
 
-```bash
-python --version
-```
-Currently Python 3.7, 3.8, 3.9, 3.10 and 3.11 are supported. Some plugins do not yet support Python 3.11.
+To simplify installing Meltano into a clean virtual environment, `pipx` can be used to install Meltano instead of `pip`. `pip` is a package installer that comes automatically with Python. [`pipx`](https://pypa.github.io/pipx/) is a wrapper around pip which cleanly installs executable Python applications (such as Meltano) into their own virtual environments.
```

Comment:
I tried clicking the first instance of `pipx` and noticed it's not linked to the pipx site. I guess that's what most folks reading this for the time would try to do

```suggestion
To simplify installing Meltano into a clean virtual environment, [`pipx`](https://pypa.github.io/pipx/) can be used to install Meltano instead of `pip`. `pip` is a package installer that comes automatically with Python. `pipx` is a wrapper around pip which cleanly installs executable Python applications (such as Meltano) into their own virtual environments.
```